### DISCUSSION TITLE
Guidance Appreciated: Upgrading lighthouse from 9.1 to 10.2

### DISCUSSION THREAD

**Discussion by RentCafeDev** [GoogleChrome/lighthouse#15167] [Q&A]
Context:
We utilize the Node API and recently upgraded lighthouse from 9.1 to 10.2. The changes were minimal and mostly converting common js to ES modules while also upgrading from Node 14 to version 16. 

With this upgrade we witnessed a 90% loss in lighthouse reporting data and in some cases a 1900% increase in connect ECONNREFUSED and LighthouseError: PROTOCOL_TIMEOUT errors. 

chrome-launcher: 14.1
Node: 16
Lighthouse: 10.2

Questions:
1) Has anyone else seen a drastic change in failed lighthouse runs when upgrading versions from 9 to 10.
2) Any advice on what I might have missed when upgrading lighthouse?

Thanks you

**Comment by connorjclark** [comment]
Can you try with the latest `chrome-launcher`? And are you on at least Node 16.16?

**Reply by RentCafeDev** [reply]
Yeah I will update chrome-launcher and get back to you once have results. Yes we are on at least Node 16.16.

**Reply by RentCafeDev** [reply]
Thank you @connorjclark. I was able to resolve the issues without upgrading chrome-launcher, so will have to hold off on that for now.
